src/components/Button/Button.test.tsx

```diff
@@ -136,5 +136,12 @@ describe('Button Component', () => {
     const button = screen.getByRole('button', { name: /Button/i })
     fireEvent.click(button)
     expect(onClick).toHaveBeenCalledTimes(1)
+    expect(onClick).toBeCalledWith(expect.objectContaining({
```

Comment:
It is needed to add this test? We should expect a MouseEvent maybe?

Reply:
I tried using expect.any(MouseEvent) but it fails. 
This to me was the closest way to test what we need from a MouseEvent. 

Would you have any recommendations on how to improve it?

Reply:
we swtiched to expect(onChange).toBeCalledWith(expect.objectContaining({ ...MouseEvent })), still not the best but should be closer to what we expect